terraform/staging_pipeline.tf

```diff
@@ -25,7 +25,7 @@ module "api_sandbox_heroku" {
     AWS_DEFAULT_REGION                 = data.aws_region.current.name
     DJANGO_ALLOWED_HOSTS               = join(",", ["api-staging.dandiarchive.org", "api.sandbox.dandiarchive.org"])
     DJANGO_CORS_ALLOWED_ORIGINS        = join(",", ["https://sandbox.dandiarchive.org", "https://gui-staging.dandiarchive.org", "https://neurosift.app"])
-    DJANGO_CORS_ALLOWED_ORIGIN_REGEXES = join(",", ["^https:\\/\\/[0-9a-z\\-]+--gui-staging-dandiarchive-org\\.netlify\\.app$"])
+    DJANGO_CORS_ALLOWED_ORIGIN_REGEXES = join(",", ["^https:\\/\\/[0-9a-z\\-]+--sandbox-dandiarchive-org\\.netlify\\.app$"])
```

Comment:
Do we not need to allow both here? Since right now we're in a state of allowing both origins (new and legacy).

Reply:
This was correct at the time, but now with #233 merged we are no longer allowing both origins.